src/components/Search.jsx

```diff
@@ -15,7 +15,6 @@ import {
   getTransaction,
   getReceiptsFromAccountPrefix } from '../services/contractUtils'
 import { useDiagramDispatch } from './DiagramState'
-import StyledButton from './StyledButton';
```

Comment:
I liked the visual indicator of the button being disabled (background color opacity lowered / changed). Can you incorporate this back in?

Reply:
and by incorporate back in, I mean just the way the button looks (active vs inactive) not necessarily the whole file.

[here is a good reference](https://getbootstrap.com/docs/4.0/components/buttons/#active-state)

Reply:
Oh sure, I had that in the css beforehand -- I think Mike removed it to add it into the styledbutton -- will add it back, thanks for the reminder